src/dashboard.js

```diff
@@ -478,6 +478,29 @@
             });
         };
 
+        /**
+        * opens content interval settings dialog for ad-sheduler app
```

Comment:
typo in comment `sheduler` -> `scheduler'

Reply:
keen eye :) fixed